index.js

```diff
@@ -1,25 +1,28 @@
 const express = require('express'),
       request = require('request'),
+      pug = require('pug'),
       morgan = require('morgan');
 
 var app = express();
 
+app.set('view engine', 'pug');
```

Comment:
we set the template engine to pug so we dont have to write it out every time as a file extension when we write out pathways to views